### ISSUE TITLE
Invertebrate Taxon data -  Bellingham 2017

### ISSUE THREAD

**Issue by helendallas** [helendallas/FRC_FBIS#84]
@Aneri16 
Please can you check you site descriptions for sites:

Rondegat | BT-CB

Also please dropbox me his thesis

**Comment by aneriswanepoel** [comment]
@helendallas 
I have added site descriptions for BT-CB - "below treatment area"
Thesis can be found on Dropbox: 
FRC_FBIS Folder for FRC\Data for Consolidation\Invertebrate Data\CFR Invertebrate data Species\FBIS data - T.Bellingan

**Comment by helendallas** [comment]
@Aneri16 thanks

**Comment by helendallas** [comment]
@Aneri16 
I have decided to extract the Bellinghan 2017 data from the FBIS ingestion file as there are several records that come up as duplicates, but I ma not sure if they are real or an artifact of biotopes.

Please can you systematically work through these for errors checking with the raw data.
The file is in the 
https://www.dropbox.com/sh/3lrrqeup1sqaxni/AADaXjW9Ujg3G7w5T6g4RMraa?dl=0

Bellingan, T.A. 2017_For revision 2020_04_16.xlsx
I have added two columns, Formula (which I used to concotenate multiple columns, and duplicates, which has the duplicates highlighted in yellow.

I included the notes as sometime this is the reason for the duplicate

**Comment by aneriswanepoel** [comment]
@helendallas I have made a start on this and from what I can tell, the "Broad biotope" column seems to have been populated quite generally (ie most of the duplicated rows have SIC) instead of according to the category specified in the raw data. It will take a day or two to fix but luckily it's pretty straightforward

**Comment by helendallas** [comment]
Please remember we have to assign the data to one of our accepted categories which are also kinda hierarchical.

Broad   Biotope
Unspecified
Mixed
Stones In Current
Stones Out Of Current
Marginal Vegetation
Aquatic Vegetation
Gravel/Sand/Mud


Specific   Biotope
Backwater
Bedrock
Cascade
Chute
Detritus
Mixed
Pool
Rapid
Riffle
Run
Slackwater
Unspecified
Waterfall


Substratum
Unspecified
Mixed
Bedrock
Boulder
Cobble
Detritus
Gravel
Pebble
Sand
Silt/Mud/Clay

You cannot have Stones In Current for Broad, then Slackwater.....for example

**Comment by aneriswanepoel** [comment]
@helendallas 
Here are the changes I have made so far (in blue) according to raw data. Bedrock in the substratum column because BRIC was the biotope in the raw data. I just want to check that I'm on the right track before I carry on... The "Specific Biotope" column only consists of Run, Pool and Unspecified for duplicate data


![Biotopes](https://user-images.githubusercontent.com/55532903/79557339-adf82700-80a2-11ea-9e85-63653d7fcba4.png)

**Comment by helendallas** [comment]
So Gravel/Sand/Mud cant have a substratum of Cobble, only gravel, sand or mud
Stones out of current, run, cobble is fine
Stones in current, run, bedrock is fine

**Comment by aneriswanepoel** [comment]
Okay thanks. Most of the changes will be along those lines

**Comment by helendallas** [comment]
Okay fine, so long as they make sense, happy for you to check each unique combination of broad, specificand substratum with me before changing

**Comment by helendallas** [comment]
From Aneri:
Just want to check in re unique biotopes - Leaf packs in current, leaf packs out of current, roots in current. What should the Broad/specific/substratum columns be?

@Aneri16 These are tricky ones and very seldom encountered. I would suggest the following, not ideal but the best without having to change the classification to accommodate these.

So classification would be:
**Leaf packs in current**
Broad: Unspecified
Specific: Detritus
Substratum: Detritus
Notes: Sampled in current

**Leaf packs out of current**
Broad: Unspecified
Specific: Detritus
Substratum: Detritus
Notes: Sampled out of current

**Roots in current**
Broad: Marginal vegetation
Specific: Run
Substratum: Unspecified
Notes: Roots sampled in current

**Comment by aneriswanepoel** [comment]
Great, thanks. And in what instance will "Pool" be the specific biotope?

**Comment by helendallas** [comment]
Pool is normally associated with the Broad biotope: stones out of current.
But if the substratum is gravel/sand/mud pool then would be Broad biotope: Gravel/Sand/Mud

**Comment by aneriswanepoel** [comment]
@helendallas okay I have finally finished checking the duplicates! What a job. Hope I got everything... I have renamed the folder with today's date. 
I extracted this data roughly a year ago and I worked according to the biotopes given in the raw data but Toni and I split Terence's PhD work between us so I have a feeling that perhaps during the combining of our spreadsheets, and then further combining with other invert spreadsheets something went wrong with the biotope data...? Just a guess. But anyway, luckily I had kept my spreadsheet from last year on my laptop so was easy to cross-reference data.
